src/api/controllers/healthcheck.ts

```diff
@@ -1,6 +1,8 @@
-const lib = require('../lib');
+import * as e from 'express';
+import lib from '../lib';
 
-exports.db = async (req, res) => {
+export const db = async (req: e.Request, res: e.Response) => {
+  res.json('working');
```

Comment:
@ManjotS are we checking Healthcheck endpoint anywhere? this would change the response.

Reply:
oh I didn't catch that it would change it. It should look as it does now unless theres a good reason to change it. (The reason is, we use the same format in all services, as set by Navdeep Singh)

Reply:
Removed it! Forgot to remove it.